be/src/exprs/bitmap_function.cpp

```diff
@@ -388,6 +388,54 @@ BigIntVal BitmapFunctions::bitmap_intersect_finalize(FunctionContext* ctx, const
     return result;
 }
 
+StringVal BitmapFunctions::bitmap_or(FunctionContext* ctx, const StringVal& src, const StringVal& dst){
+    RoaringBitmap bitmap;
```

Comment:
To make code clear, you can do as follow

```
if (lhs.is_null || rhs.is_null) {
    return StringVal::null();
}
```

And then there is no need to check src.is_null again.

Reply:
done

Reply:
done